### DISCUSSION TITLE
Current limit setting

### DISCUSSION THREAD

**Discussion by chucky-whiskers** [iNavFlight/inav#8665] [Q&A] [answered]
## Current Behavior
 I am having trouble finding a way to limit throttle output for higher voltage batteries 

## Desired Behavior
I would like to see a way to use battery profiles to limit current to motors in order to be able to fly motors and ESC's rated for low voltage on a higher voltage battery

## Suggested Solution
 the solution is a simple throttle attenuation modifier that can be applied within each battery profile 

## Who does this impact? Who is this for?
this is for anyone wanting to fly long range and increase efficiency while lowering amp draw 

## Additional context
Add any other context or links about the feature request here.

**Comment by DzikuVx** [comment, marked as answer]
Take a look here https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#limit_burst_current and a few setings below it. You can both limit current and power

**Reply by chucky-whiskers** [reply]
Thanks for the link! I think all I needed was right there. It would be nice to have a simple menu to change these parameters, though. This is probably not a small project, as it would require a logic based algorithm. If a person could select their actual nominal battery voltage and the nominal voltage they would like to emulate via current limits through the MOSFET(s), it would be possible to run a 6S LiPo to power a plane (or other craft) that normally runs a 3S LiPo. The current sensor does a remarkable job recognizing actual voltage. However, being able to select a lower "target voltage behavior" would enable longer battery life and lower heat loss. Do you think that it is unwise to do such a thing, since some motors are not rated for high voltage input? A MOSFET will still apply full voltage to the ESC in small bursts, right? High voltage, low amp draw is the key to long range flight.
